plugins/source/cloudflare/plugin/plugin.go

```diff
@@ -0,0 +1,48 @@
+package plugin
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/cloudflare/client"
+	"github.com/cloudquery/cloudquery/plugins/source/cloudflare/services"
+	"github.com/cloudquery/plugin-sdk/plugins"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+const exampleConfig = `
+# Required. OAuth token to authenticate with Heroku API
+token: <token>
+
+## Optional. GRPC Retry/backoff configuration, time units in seconds. Documented in https://github.com/grpc/grpc/blob/master/doc/connection-backoff.md
+# backoff_base_delay: 1
+# backoff_multiplier: 1.6
+# backoff_max_delay: 120
+# backoff_jitter: 0.2
+# backoff_min_connect_timeout = 0
+## Optional. Max amount of retries for retrier, defaults to max 3 retries.
+# max_retries: 3
```

Comment:
This example config needs to be updated to match the expectations for cloudflare. (and the name in the comment changed from `Heroku` -> `Cloudflare` 😉 )

I also saw @yevgenypats remove these backoff options from the gcp plugin in recent PR. I don't know the reasoning, but we should probably remove it here and from the Heroku plugin as well.

Reply:
Good catch, agree we need to remove it here as well. I dont think the backoff stuff in GCP ever worked at least in the expected way and we dont know if they were use so rather then introducing options that confuse users I would prefer for now to remove it and add it later if it will be requested.

Reply:
oops yeah, I meant to use ExampleConfig but then copied your plugin.

Reply:
I like your way of putting `Example()` as a method on the `Spec` struct - makes it less likely that you'll forget to update the example when the config changes. We should probably adopt that as a standard.